### PR TITLE
Fixes #705 fixed changes UI of activity_my_trip_info

### LINKED ISSUE #705: Trip Details Screen ui seems not good
Trip Screen UI not looking good

trip_details_ui

### PR DESCRIPTION
**Pull request by AminullahTajMuhammad**
## Description

Changed the UI of Trip Detail Screen

## Before Changes.
trip_details_ui

## After Changes
changes_ui_trip

Fixes #705 

## Type of change
Just put an x in the [] which are valid.
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes.
- [x] `./gradlew assembleDebug assembleRelease`
- [x] `./gradlew checkstyle`

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Swati4star** [comment]
- Reduce the size of "travelling with someone..."
- reduce space between trip name and trip date
- why so much space after "travelling with someone.."?
- In specs, after "trip type:" there was supposed to be an icon showing trip type. Writing it in next line is becoming so confusing UX.

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripInfoActivity.java [hunk 1]
```diff
@@ -628,17 +633,19 @@ public void onResponse(Call call, final Response response) throws IOException {
                                 addNewFriend.setVisibility(GONE);
                                 friendEmail.setVisibility(GONE);
                                 noFriendTitle.setVisibility(VISIBLE);
-                                noFriendTitle.setTypeface(null, Typeface.BOLD);
+//                                noFriendTitle.setTypeface(null, Typeface.BOLD);
```
Comment (Swati4star) [inline Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripInfoActivity.java]: don't comment. if you want to remove, remove it

File: Android/app/src/main/res/layout/activity_my_trip_info.xml [hunk 2]
```diff
@@ -29,7 +29,7 @@
                 android:contentDescription="@string/empty_description"
                 android:scaleType="centerCrop"
                 android:src="@drawable/placeholder_image"
-                android:visibility="gone"
+                android:visibility="visible"
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/activity_my_trip_info.xml]: why this change?

File: Android/app/src/main/res/layout/content_my_trip_info.xml [hunk 3]
```diff
@@ -44,48 +43,44 @@
                     android:layout_weight="1"
                     android:fontFamily="sans-serif-bold"
                     android:padding="10dp"
+                    android:text="Chennai"
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/content_my_trip_info.xml]: don't add default text

File: Android/app/src/main/res/layout/content_my_trip_info.xml [hunk 4]
```diff
@@ -44,48 +43,44 @@
...
+
                 <EditText
                     android:id="@+id/trip_name"
-                    android:layout_width="318dp"
+                    android:layout_width="wrap_content"
                     android:layout_height="match_parent"
                     android:background="@android:color/transparent"
                     android:cursorVisible="false"
                     android:ellipsize="end"
+                    android:text="My Trip"
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/content_my_trip_info.xml]: remove default text

File: Android/app/src/main/res/layout/content_my_trip_info.xml [hunk 5]
```diff
@@ -44,48 +43,44 @@
...
             <RelativeLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content">
 
+                <TextView android:id="@+id/label_trip_name"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:padding="10dp"
+                    android:textSize="20sp"
+                    android:text="Trip Name: "
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/content_my_trip_info.xml]: these should go in strings.xml file

File: Android/config/quality/checkstyle_config.xml [hunk 6]
```diff
@@ -4,7 +4,6 @@
     "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">
 
 <module name="Checker">
-
```
Comment (Swati4star) [inline Android/config/quality/checkstyle_config.xml]: why this change?

Comment (AminullahTajMuhammad) [reply Android/config/quality/checkstyle_config.xml]: there is nothing change just remove a simple space.

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripInfoActivity.java (hunk 1 above)
Comment (AminullahTajMuhammad) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripInfoActivity.java]: Done

File: Android/app/src/main/res/layout/activity_my_trip_info.xml (hunk 2 above)
Comment (AminullahTajMuhammad) [reply Android/app/src/main/res/layout/activity_my_trip_info.xml]: It was changed for just testing. Changes in next new commits

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 3 above)
Comment (AminullahTajMuhammad) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: Done

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 4 above)
Comment (AminullahTajMuhammad) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: Done

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 5 above)
Comment (AminullahTajMuhammad) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: Done

**Comment by AminullahTajMuhammad** [comment]
#### why so much space after "traveling with someone.."
Because there is a listView of friend list. That's why I haven't change this

#### In specs, after "trip type:" there was supposed to be an icon showing trip type. Writing it in next line is becoming so confusing UX.
So, can I remove the text of public/private and also add icon instead of trip type button

**Comment by Swati4star** [comment]
if there are no items in list view -> then maybe hide it. I think the min height of that list is too much. We should reduce it.

Remove the text "trip type". Move the description "This is a public trip...." before the "Make private" button. Seems like, you have increased the size of "This is a public trip...." also. Reduce it.

File: Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripInfoActivity.java (hunk 1 above)
Comment (Swati4star) [reply Android/app/src/main/java/io/github/project_travel_mate/mytrips/MyTripInfoActivity.java]: not done. It's still here

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 3 above)
Comment (Swati4star) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: Whichever review comment you have marked as "Done" are not. Please check

File: Android/config/quality/checkstyle_config.xml (hunk 6 above)
Comment (Swati4star) [reply Android/config/quality/checkstyle_config.xml]: then, remove this file from commit

Comment (prabhakar267) [reply Android/config/quality/checkstyle_config.xml]: @AminullahTajMuhammad if you are not making any relevant change to some file, you shouldn't add it to your git staging and commits since it adds you to the file's contributor list making it hard to track in future who worked on what

**Comment by prabhakar267** [comment]
@AminullahTajMuhammad Any updates?

**Comment by AminullahTajMuhammad** [comment]
@prabhakar267 I am working on it.

**Comment by AminullahTajMuhammad** [comment]
@Swati4star and @prabhakar267 I just added a new commit. Please check it.

File: Android/app/src/main/res/layout/activity_my_trip_info.xml (hunk 2 above)
Comment (Swati4star) [reply Android/app/src/main/res/layout/activity_my_trip_info.xml]: You havent changed it yet

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 3 above)
Comment (Swati4star) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: ??

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 5 above)
Comment (Swati4star) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: Not done @AminullahTajMuhammad

File: Android/app/src/main/res/layout/activity_my_trip_info.xml (hunk 2 above)
Comment (AminullahTajMuhammad) [reply Android/app/src/main/res/layout/activity_my_trip_info.xml]: Done in new commit

**Comment by AminullahTajMuhammad** [comment]
@Swati4star Added all changes. Please check it.

File: Android/app/src/main/res/layout/content_my_trip_info.xml [hunk 7]
```diff
@@ -105,22 +98,49 @@
                     android:padding="10dp"
                     android:paddingBottom="13dp"
                     android:src="@drawable/ic_edit_black_24dp" />
+
+                <TextView android:id="@+id/label_trip_date"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_below="@+id/trip_name"
+                    android:visibility="gone"
+                    android:text="Trip Date: "
```
Comment (Swati4star) [inline Android/app/src/main/res/layout/content_my_trip_info.xml]: dont add constant string in strings.xml file

File: Android/config/quality/checkstyle_config.xml (hunk 6 above)
Comment (Swati4star) [reply Android/config/quality/checkstyle_config.xml]: remove this file from commit

**Comment by AminullahTajMuhammad** [comment]
@Swati4star Check new changes

**Comment by Swati4star** [comment]
Please check my previous review comments. They are still not resolved.

**Comment by prabhakar267** [comment]
Closing this due to inactivity, @AminullahTajMuhammad feel free to reopen if you address the comments.

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 7 above)
Comment (AminullahTajMuhammad) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: Can I add this text in string.xml file or not.

**Comment by AminullahTajMuhammad** [comment]
https://github.com/project-travel-mate/Travel-Mate/pull/728#discussion_r335295094

File: Android/app/src/main/res/layout/content_my_trip_info.xml (hunk 7 above)
Comment (Swati4star) [reply Android/app/src/main/res/layout/content_my_trip_info.xml]: Yes. add constants to strings.xml file. Not directly in layout files.

**Comment by AminullahTajMuhammad** [comment]
@prabhakar267 Sorry for late replying. I am busy with exams. I don't understand https://github.com/project-travel-mate/Travel-Mate/pull/728#discussion_r335295094. Should I add this text in string.xml or not?

**Comment by prabhakar267** [comment]
@AminullahTajMuhammad It shouldn't be adding strings like this directly in the layout files instead add them to strings.xml and reference it here.
